crates/stc_ts_type_checker/tests/tsc.rs

```diff
@@ -216,6 +216,9 @@ fn create_test(path: PathBuf) -> Option<Box<dyn FnOnce() + Send + Sync>> {
 
     if let Ok(errors) = load_expected_errors(&path) {
         for err in errors {
+            if err.code == "TS1318" {
```

Comment:
I think this line is doing a thing identical to the line below

Reply:
The line below ignores the test `return None`. This line prevents the test from being ignored.

Reply:
My bad, thanks!